versions/blockspacerace_versions.js

```diff
@@ -1,9 +1,9 @@
 const blockspaceraceVersions = Object.freeze({
-  "app-latest-tag": "v0.13.0",
-  "app-latest-sha": "6ba60e18fec27d3e3a42a71be082e255913cf4cf",
-  "core-latest-tag": "v1.18.0-tm-v0.34.27",
-  "core-latest-sha": "9bfa856909861081d2bbc715ee00ed56e9fc43bb",
-  "node-latest-tag": "v0.9.2",
-  "node-latest-sha": "4d31ad4a537e41b5ac588e14cf807904abbb8d8a",
+  "app-latest-tag": "v0.13.2",
```

Comment:
👍  https://github.com/celestiaorg/celestia-app/releases/tag/v0.13.2 is what should be used for ITN hard fork